torch/cuda/graphs.py

```diff
@@ -187,7 +188,12 @@ def __exit__(self, exc_type, exc_value, traceback):
 
 
 def make_graphed_callables(
-    callables, sample_args, num_warmup_iters=3, allow_unused_input=False, pool=None
+    callables,
+    sample_args,
+    num_warmup_iters=3,
+    allow_unused_input=False,
+    pool=None,
+    disable_autocast_bwd=None,
```

Comment:
Default this to True now, no?

Reply:
I think @eellison wanted this to default to `None`?